File: ERPServer/Shared/Shared/Entities/Setup/ProductTrackingSearchDTO.cs
using Data.Entities.Shared;
using Shared.Entities.Shared;
using Shared.Enums;
using System;


namespace Shared.Entities.Setup
{
    public class ProductTrackingSearchDTO : Paging
    {
        public long ProductId { get; set; }
        public string Date { get; set; }
        public ProductProcessTypeEnum? ProductProcessTypeId { get; set; }
    }
}

File: erp-app/src/app/modules/setup/models/product-tracking-search.dto.ts
import { ProductProcessTypeEnum } from "src/app/shared/enums/product-process-type.enum";
import { PagingDTO } from "src/app/shared/models/paging-dto";

export class ProductTrackingSearchDTO extends PagingDTO {
    productId: number;
    date: string;
    productProcessTypeId: ProductProcessTypeEnum;
}